src/modules/QtQuick/Animation.js

```diff
@@ -1,19 +1,21 @@
-QmlWeb.registerQmlType({
-  module: "QtQuick",
-  name: "Animation",
-  versions: /.*/,
-  baseClass: "QtQml.QtObject",
-  enums: {
+QmlWeb.registerQmlType(class {
+  static module = "QtQuick";
+  static element = "Animation";
```

Comment:
why did you rename that property to element? I think "element" is something you shouldn't use as a name for (almost) anything, because it's too vague... What's wrong about `name`?